plugins/mcreator-localization/help/default/command/server_type.md

```diff
@@ -0,0 +1,9 @@
+This option allow you to determine on which type(s) of server the command can be used.
+* **Both**: The command can be used everywhere. THis is the normal behaviour (e.g. `/give`.
+* **Multi players only**: The command will be available on multi players servers only (e.g. `/ban`)

```

Comment:
Minecraft call this "Multiplayer" and not multi players, so the naming should be streamlined, same for Singleplayer